### ISSUE TITLE
Fix warning related to deprecation of vscode.previewHtml

### ISSUE THREAD

**Issue by cbjuan** [Qiskit/qiskit-vscode#62]
### What is the expected enhancement?
When executing the extension, the debug terminal raises this warning:
```
Extension 'qiskit.qiskit-vscode' uses the 'vscode.previewHtml' command which is deprecated and will be removed. Please update your extension to use the Webview API: https://go.microsoft.com/fwlink/?linkid=2039309
```

We should address the changes in the API to avoid a future error in the extension.

**Comment by cbjuan** [comment]
More info: https://github.com/Microsoft/vscode/issues/62630

**Comment by cbjuan** [comment]
Fixed in #79
